invenio_cli/commands/services.py

```diff
@@ -34,21 +34,21 @@ def ensure_containers_running(self):
             project_shortname=project_shortname,
         )
         return ProcessResponse(
-                output="Containers start and checkup run.",
-                status_code=0,
+            output="Containers started and healthy.",
```

Comment:
comment: this is not 100% true since ES will still show a warning due to not being in cluster mode (status=yellow).

Reply:
close enough :smile_cat: